src/matrix2graph.jl

```diff
@@ -0,0 +1,50 @@
+include("custom_graph.jl")
+
+"""
+        matrix2graph(SparseMatrix)
+
+A utility function to generate a graph from input
+sparse matrix, columns are represented with vertices
+and 2 vertices are connected with an edge only if
+the two columns are not mutually orthogonal.
+"""
+function matrix2graph(SparseMatrix::Array{Int64,2})
```

Comment:
You'll want this to specialize on `SparseMatrixCSC{T}`. In that, there is 

```julia
using SparseArrays
findnz(sprand(100,100,0.01))
```

to get the non-zero value locations

Reply:
Used SparseArrays in the code instead of simple Arrays{Int64}. The inbuilt functions also allowed me to simplify the code a lot more without using all the logical comparisons that were being done in the previous version.